### ISSUE TITLE
Just FYI, Why not start with trinitytbc?

### ISSUE THREAD

**Issue by coolzoom** [FALL1N1/TBC_New#1]
**Description:**

Hello
Interesting project, but I have a question why not just fork trinitytbc and start from there?

https://github.com/trinitytbc

It is also based on sunstrider and already ported a lot scripts.

You probably already know it and have your own decision. This is just FYI just in case you don’t know it.

CHANGEME Description of the problem or issue here.

**Current behaviour:**

CHANGEME Tell us what happens.
If this is a crash, post the crashlog (upload to https://gist.github.com/).

**Expected behaviour:**

CHANGEME Tell us what should happen instead.

**Steps to reproduce the problem:**

1. CHANGEME Step 1 include entries of affected creatures / items / quests with a link to the relevant wowhead page.  
2. Step 2
3. Step 3

BE VERY SPECIFIC HERE. Even if the problem seems general, do tell which spells did you use, on what creature, in which conditions. (Very often the problem is only happening in a specific context)

**rev. hash/commit:** 

CHANGEME
If you're testing on the public server, do a `.server info` in game and copy/screenshot the first line.  
If compiling the server yourself, copy the first line of the `worldserver` output.  
For example: Sunstrider rev. 0000000000 2000-01-09 11:31:41 +0100 (my branch) (Win64, RelWithDebInfo, Static) (worldserver)

**Operating system and compiler:** CHANGEME OS
Delete this section if you're testing on the public servers.

**Comment by FALL1N1** [comment]
Hello!

I'm backporting the scripts from SunwellCore's 3.3.5a release as they are better in some cases and provide better content, from a fast look in the creature_template's ScriptName column I saw that the value for some of the NPCs is from Sunwell's script name.
Either way, I did not know about the trinitytbc fork, I might merge it later, thank you!

About the template thank you, I will update it whenever I have free time, appreciated!
